package.json

```diff
@@ -3,18 +3,24 @@
   "version": "0.0.5",
   "description": "TiDB Serverless driver",
   "main": "dist/index.js",
```

Comment:
main usually point to the commonjs file https://stackoverflow.com/questions/68572936/what-is-the-difference-between-main-and-module-vs-exports-in-package-json